### PR TITLE
bump httpclient to 4.5.2 and httpcore to 4.4.4

### PR DESCRIPTION
**Pull request by bchen-talend**
## What changes were proposed in this pull request?
bump httpcore version to make elasticserach component working on SJS

### CODE REVIEW

**Review by pradasouvanlasy — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -504,7 +505,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpcore</artifactId>
-        <version>${commons.httpclient.version}</version>
```
Comment (amarouni) [inline pom.xml:507]: @bchen-talend Why you're removing this dependency ?

**Review by amarouni — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (pradasouvanlasy) [reply pom.xml:507]: He's not removing it, httpclient and httpcore used to be set at the same version number. Now they have different versions

Comment (pradasouvanlasy) [reply pom.xml:507]: See https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient/4.5.2 which depends of httpcore 4.4.4

Comment (amarouni) [reply pom.xml:507]: Ohh my bad sorry

**Review by amarouni — APPROVED** [review]
LGTM

**Comment by pradasouvanlasy** [comment]
Later on, we will build a spark-assembly from there, publish it into Nexus and update SJS accordingly.
QA will have to run tests to see if we did not break the existing components.

**Pull request merged**